fms_core/resources/sample_update.py

```diff
@@ -22,8 +23,9 @@ class SampleUpdateResource(GenericResource):
                       widget=ForeignKeyWidget(Container, field='barcode'))
     coordinates = Field(attribute='coordinates', column_name='Coord (if plate)')
     # fields that can be updated on sample update
-    # new volume
+    # volume
     volume_history = Field(attribute='volume_history', column_name='New Volume (uL)')
+    volume_history_delta = Field(attribute='volume_history_delta', column_name='Delta Volume (uL)')
```

Comment:
since this attribute doesn't exist on the real models, this may cause an issue when trying to export from the admin interface - may be good to check this

Reply:
Thanks i'll test this. If this crashes things, just removing the attribute should fix this right ?